src/hb-open-type.hh

```diff
@@ -604,7 +604,7 @@ struct ArrayOf
   hb_array_t<Type> sub_array (unsigned int start_offset, unsigned int *count = nullptr /* IN/OUT */)
   { return as_array ().sub_array (start_offset, count); }
 
-  bool serialize (hb_serialize_context_t *c, unsigned int items_len)
+  hb_success_t serialize (hb_serialize_context_t *c, unsigned items_len)
```

Comment:
As this initially was because of https://github.com/harfbuzz/harfbuzz/commit/57b7de032f60d0499ae2debb293d0f8456acfdfb will gradually other methods also.